docs/guides/dev_guide.md

```diff
@@ -188,20 +198,20 @@ More information: [Protocol-rfc#0018 - Transaction Payload](https://github.com/l
 `SignedTransaction` is a payload type that is used to transfer value-based messages as `UTXO`. It changes the ledger state as old `outputs` are being spent (replaced) and new `outputs` are being created.
 
 Each `SignedTransaction` includes the following set of information:
-* `inputs` - a list of valid `outputs` that should be used to fund the given transaction. Outputs are uniquely referenced via previous `transaction_id` and inner `index`. At least one output has to be given with enough balance to source all `outputs` of the given message
-* `outputs` - a list of IOTA address(es) and related amount(s) the input `outputs` should be split among. Based on this information, new `UTXO` outputs are being created
-* `unlock_blocks` - it includes a transaction signature(s) (currently based on `Ed25519` scheme) that proves token ownership based on a valid seed
+* `inputs` - a list of valid `outputs` that should be used to fund the given transaction. Outputs are uniquely referenced via previous `transaction_id` and inner `index`. At least one output has to be given with enough balance to source all `outputs` of the given message.
+* `outputs` - a list of IOTA address(es) and related amount(s) the input `outputs` should be split among. Based on this information, new `UTXO` outputs are being created.
+* `unlock_blocks` - it includes a transaction signature(s) (currently based on `Ed25519` scheme) that proves token ownership based on a valid seed.
 
-## Dust protection
+## Dust Protection
 
 Since IOTA is feeless and has the ability to send microtransactions, attackers could use this to spam the network with very low value transactions, which we call dust. To avoid this, we only allow microtransaction below 1Mi (dust) of IOTA tokens to another address if you already have at least 1Mi as a dust allowance output on that address. The number of allowed dust outputs on an address is the amount of the dust allowance outputs divided by 100,000 and rounded down, i.e. 10 outputs for each 1 Mi deposited, with a maximum of 100 dust outputs in total.
```

Comment:
This last sentence seems a bit long.  Perhaps we could the i.e. ... to a new sentence.